internal/provider/provider.go

```diff
@@ -155,6 +155,8 @@ func WithGroupIssues() Ops {
 func WithAppVulns() Ops {
 	return func(provider *Options) error {
 		provider.flags = append(provider.flags, "--app-vulns")
+		// We started with a default depth value of 2
```

Comment:
This limits the "maximum depth" to search? Would setting higher values cause a large overhead? (wondering if there would be downsides to setting it higher).

Reply:
See the internal Slack thread which explains this choice

Reply:
we don't know exactly, but we know that it works with the default value which is 1, so we thought 2 is better :-)

Reply:
As long as `2` doesn't mean "skip those that are only nested `1` level deep" 😬 😅